tests/slow.haskell

```diff
@@ -353,6 +353,7 @@ tests/specs/erc20/ds/transfer-success-2-spec.k
 tests/specs/erc20/hkg/transferFrom-failure-1-spec.k
 tests/specs/erc20/hkg/transferFrom-success-1-spec.k
 tests/specs/examples/erc20-spec.md
+tests/specs/examples/erc721-spec.md
```

Comment:
This should probably just be in `tests/failing.haskell` instead. Is it actually slow? (passing on your machine, timing out on CI) If it's failing on CI, let's just mark it failing.

Reply:
Done.